modules/preview-react/breadcrumbs/stories/Breadcrumbs.stories.mdx

```diff
@@ -0,0 +1,216 @@
+
+<Meta title="Preview/Breadcrumbs/React" component={Breadcrumbs} />
+
+# Canvas Kit Breadcrumbs
+
+Breadcrumbs allow users to keep track and maintain awareness of their location as they navigate
+through pages, folders, files, etc.
+
+[> Workday Design Reference](https://design.workday.com/components/navigation/breadcrumbs)
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-preview-react
+```
+
+## Usage
+
+Breadcrumbs is a composable component that provides some flexibility in it's structure. There are
+two main variations: standard list and collapsible list. We'll discuss the standard list first. All
+subcomponents are described in more detail in the [Components](#components) section below.
+
+### Standard List Example
+
+`Breadcrumbs.List` is a styled unordered list `<ul>` with no additional behaviors. Use this when you
+are not concerned about collapsing the list of breadcrumb items into a dropdown menu at a specified
+width.
+
+Accessibility is built into Breadcrumbs components in a way that allows you to create an inclusive
+experience with little additional configuration. The only ARIA label you'll need to add is the
+`aria-label` for `Breadcrumbs.Nav` as seen in the example. And all links require `href`s to be
+properly identified. All other accessible attributes are baked into the components.
+
+<ExampleCodeBlock code={StandardList} />
+
+### Collapsible List Example
+
+`Breadcrumbs.CollapsibleList` is built on top of `Breadcrumbs.List`. It provides additional
+functionality to collapse list items into a dropdown menu at a specified width.
+
+In addition to adding `aria-label` to `Breadcrumbs.Nav` and identifying all links with an `href`,
+you'll need to add `buttonAriaLabel` to `Breadcrumbs.CollapsibleList`.
+
+<ExampleCodeBlock code={CollapsibleList} />
+
+### Right-to-Left Example
+
+Breadcrumbs has bidirectional support out of the box. That means outside of setting the content
+direction in your application's Canvas theme, you don't need to do anything else to make it work.
+All you need to supply is the translated text for items and ARIA labels.
+
+<ExampleCodeBlock code={RTLCollapsibleList} />
+
+## Components
+
+### Breadcrumbs.Nav
+
+#### Usage
+
+`Breadcrumbs.Nav` a styled `<nav>` element. It has no intrinsic state or behaviors.
+
+#### Props
+
+<ArgsTable of={Breadcrumbs.Nav} />
+
+> Note: This component also supports
+> [all native HTMLElement props](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes).
+
+---
+
+### Breadcrumbs.List
+
+#### Usage
+
+`Breadcrumbs.List` component is a styled, unordered list (`<ul>`) with no additional behaviors. Use
+this when you're not concerned about collapsing the list of breadcrumb items into a dropdown menu at
+a specified width.
+
+#### Props
+
+`Breadcrumbs.List` has no component-specific props. However, it supports
+[all native `<ul>` element props](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul#Attributes).
+
+---
```

Comment:
@jamesfan

How do you feel about dividers between components and subcomponents? I didn't see any on `Card`, but I added them here to break up the content a bit. I don't have a strong preference, but we should probably pick one and stick to it.